src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -2610,7 +2610,7 @@ def aks_agentpool_delete(cmd, client, resource_group_name, cluster_name,
         raise CLIError("Node pool {} doesnt exist, "
                        "use 'aks nodepool list' to get current node pool list".format(nodepool_name))
 
-    return sdk_no_wait(no_wait, client.begin_delete, resource_group_name, cluster_name, nodepool_name)
+    return sdk_no_wait(no_wait, client.begin_delete, resource_group_name, cluster_name, nodepool_name, ignore_pod_disruption_budget=ignore_pod_disruption_budget)
```

Comment:
```suggestion
    from azure.cli.core.cloud import get_active_cloud
    active_cloud = get_active_cloud(cmd.cli_ctx)
    if active_cloud.profile != "latest":
        return sdk_no_wait(
            no_wait,
            client.begin_delete,
            resource_group_name,
            cluster_name,
            nodepool_name,
        )
    else:
        return sdk_no_wait(
            no_wait,
            client.begin_delete,
            resource_group_name,
            cluster_name,
            nodepool_name,
            ignore_pod_disruption_budget=ignore_pod_disruption_budget,
        )
```
something like this can be used as a workaround to support both the old profile with 2020-11-01 as the default API version and the latest profile with the latest API version.

Please move `from azure.cli.core.cloud import get_active_cloud` to the top of this file

Reply:
Added. It seems it still doesn't work.